### PR TITLE
Update dockerfile_stage3

### PR DESCRIPTION
**Pull request by redoracle**
docker improvements

### CODE REVIEW

File: files/docker/node/dockerfile_stage3 [hunk 1]
```diff
@@ -57,24 +56,13 @@ WORKDIR /home/guild
...
-    && echo "alias cncli=$CNODE_HOME/scripts/cncli.sh" >> ~/.bashrc \
-    && echo "export PATH=/nix/var/nix/profiles/per-user/guild/profile/bin:/nix/var/nix/profiles/per-user/guild/profile/sbin:/opt/cardano/cnode/scripts:/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/home/guild/.cabal/bin"  >> ~/.bashrc
-
-# INSTALL DEPS  
-RUN /nix/var/nix/profiles/per-user/guild/profile/bin/nix-env -i python3 systemd libsodium tmux jq ncurses libtool autoconf git wget gnupg tcptraceroute util-linux less openssl \
-    && /nix/var/nix/profiles/per-user/guild/profile/bin/nix-channel --update \
-    && /nix/var/nix/profiles/per-user/guild/profile/bin/nix-env -u --always \
-    && /nix/var/nix/profiles/per-user/guild/profile/bin/nix-collect-garbage -d \
-    && sudo rm /nix/var/nix/profiles/per-user/guild/profile/bin/nix-*
+    && echo "alias cnclis=$CNODE_HOME/scripts/cncli.sh" >> ~/.bashrc \
```
Comment (TrevorBenson) [inline files/docker/node/dockerfile_stage3:64]:
```suggestion
    && echo "alias cncli=$CNODE_HOME/scripts/cncli.sh" >> ~/.bashrc \
```

**Review by TrevorBenson — CHANGES_REQUESTED** [review]
LGTM except the cncli alias.

**Review by TrevorBenson — APPROVED** [review]

**Comment by redoracle** [comment]
> LGTM except the cncli alias.

Why you didn't fix before merging?

**Comment by TrevorBenson** [comment]
Strange, I had suggested a change to fix the s, and I clicked confirm on it
before I chose to merge. I presumed it was committed and included.

**Pull request merged**